lib/shared/bucketing-assembly-script/assembly/types/platformData.ts

```diff
@@ -4,7 +4,6 @@ import { getStringFromJSON } from '../helpers/jsonHelpers'
 export class PlatformData extends JSON.Obj {
     platform: string
     platformVersion: string
-    deviceModel: string
```

Comment:
Why did you remove it from here?

Reply:
because deviceModel isn't platform data, it's user data. In a backend context the "device" is associated with the current user being processed, it's not a platform-level thing